src/ImageSharp/Processing/Contrast/HistogramEqualizationExtension.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace SixLabors.ImageSharp.Processing.Contrast
```

Comment:
`SixLabors.ImageSharp.Processing.Normalization`?

Reply:
i have changed that